ocaml/idl/datamodel.ml

```diff
@@ -6659,6 +6663,16 @@ let pool_create_VLAN = call
     ~allowed_roles:_R_POOL_OP
     ()
 
+let pool_management_reconfigure = call
+    ~name:"management_reconfigure"
+    ~in_oss_since:None
+    ~in_product_since:rel_falcon
+    ~params:[
+             Ref _network, "network", "network to which all hosts management to be reside in the Pool";
```

Comment:
I have trouble parsing this. Do you mean one of these? 

* "the network where all management interfaces reside"
* "the network of all management interfaces"
* "the shared network of all management interfaces in the pool"